cli/asc.js

```diff
@@ -298,43 +317,33 @@ exports.main = function main(argv, options, callback) {
           return p;
         });
       }
-      args = optionsUtil.merge(exports.options, args, asconfig.options);
+      args = optionsUtil.merge(exports.options, asconfig.options, args);
```

Comment:
This is the wrong order. The asconfig options need to be overridden by cli args.

The options.merge function puts the parent as the third parameter, preferring the second argument.

Did I miss something or mis-implement this algorithm?

Reply:
You're right.  Just added more tests and sure enough it ignores the CLI args.  The issue was that the defaults for the args was being set before the config properties were being added.  So if you treat `args` as the current config it will prioritize the defaults.  I just changed it to not add the defaults right away so we can get just the args passed and things seem to work.